scarb/src/ops/resolve.rs

```diff
@@ -59,13 +59,50 @@ impl WorkspaceResolve {
         &self,
         package_id: PackageId,
         target_kind: &TargetKind,
-    ) -> Vec<Package> {
+    ) -> Result<Vec<Package>> {
         assert!(self.packages.contains_key(&package_id));
-        self.resolve
+        let dependencies = self
+            .resolve
             .package_dependencies_for_target_kind(package_id, target_kind)
             .iter()
             .map(|id| self.packages[id].clone())
-            .collect_vec()
+            .collect_vec();
+
+        let reexported = dependencies
```

Comment:
```suggestion
        let re_exported = dependencies
```
Let's update it everywhere across the codebase, including the docs, just so all related elements are easily searchable.